### PR TITLE
Add task started and task resolved columns

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: src/views/Workers/WorkerManager.js [hunk 1]
```diff
@@ -34,6 +34,19 @@ export default class WorkerManager extends React.PureComponent {
     }
   }
 
+  loadStatus = async (taskId) => {
+    if (!taskId) {
+      return {};
+    }
+
+    const { status } = await this.props.queue.status(taskId);
+
```
Comment (eliperelman) [inline src/views/Workers/WorkerManager.js]: Remove extra blank line.

File: src/views/Workers/WorkerManager.js [hunk 2]
```diff
@@ -34,6 +34,19 @@ export default class WorkerManager extends React.PureComponent {
...
+    const runs = status.runs.length - 1;
+    const lastClaimStarted = status.runs[runs].started;
```
Comment (eliperelman) [inline src/views/Workers/WorkerManager.js]: These can be inlined into the returning object.

File: src/views/Workers/WorkerManager.js [hunk 3]
```diff
@@ -94,26 +111,30 @@ export default class WorkerManager extends React.PureComponent {
         <Table className={styles.workersTable} responsive condensed={true} hover={true}>
           <thead>
             <tr>
-              <th>Worker ID</th>
-              <th>Most Recent Task</th>
-              <th>First Claim</th>
-              <th>Status</th>
+              <th title="Worker ID">Worker ID</th>
+              <th title="Most Recent Task">MRT</th>
```
Comment (eliperelman) [inline src/views/Workers/WorkerManager.js]: I find the phrase MRT to be vague. Is there another way we can phrase this to be succinct without needing an abbreviation?

Comment (djmitche) [reply src/views/Workers/WorkerManager.js]: I do too -- I opened up this PR to figure out what the heck an MRT was!  Maybe
```
Most Recent Task       Task Started      Task Resolved
```

**Pull request merged**